src/main/java/item/model/Item.java

```diff
@@ -63,6 +76,11 @@ public Builder setPicture(String picture) {
       return this;
     }
 
+    public Builder setId(int id) {
+      this.id=id;
+      return this;
+    }
+
     public Builder setRating(double rating) {
```

Comment:
Using builder pattern, great!
but where do you build an Item?

Reply:
At the beginning i was going to use it, but i changed the way i wanted to deal with the requests, so up until now it is being used nowhere